tests/integration/basic_c.cpp

```diff
@@ -164,4 +181,35 @@ TEST_F(MemgraphConnection, InsertAndRetriveFromMemegraph) {
   ASSERT_EQ(rows, 1);
   ASSERT_EQ(status, 0);
   ASSERT_EQ(mg_session_commit_transaction(session, &result), 0);
+
+  {
+    ASSERT_EQ(mg_session_run(session,
```

Comment:
Isn't this for unit test and not for integration/e2e?

Reply:
It is, but I would like to have something like this (probably more, with more stuff tested) on the integration/end2end level because these tests will be called on each platform. It's going to be required that all integration tests pass on each supported platform. Not all unit tests work on every platform (we have to port everything which will take more time / it's not critical at this point, e.g., socketpair is tricky ON_WINDOWS). I mean, I can move this in the `tests/unit` but I would like to keep this test as it is now (or very similar).

Reply:
Oh yeah, forgot we won't have unit tests on all platforms for now.
Could you just add a comment why this isn't in unit folder then?